yt_idv/shaders/display_border.frag.glsl

```diff
@@ -0,0 +1,13 @@
+in vec2 UV;
+
+out vec4 color;
+
+void main(){
+   color = texture(fb_tex, UV);
+   color.a = 1.0;
+   vec2 d = abs(UV - vec2(0.5));
+   if(0.5 - max(d.x, d.y) < draw_boundary) {
+      color = vec4(boundary_color);
+   }
+   gl_FragDepth = texture(db_tex, UV).r;
```

Comment:
you've got an indentation of 3 spaces instead of 4 on all these lines (all the other shaders use 4 spaces)... I don't care but maybe you do :)

Reply:
I noticed at least one other shader with only 3 spaces in the indentation level, so let's keep this as is and re-format them all to match at some point if we want.